src/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import { OctokitOptions } from "@octokit/core/dist-types/types.d";
+
+type LimitHandler = (retryAfter: number, options: any, octokit: any) => void;
+
+export interface ThrottlingOptions extends OctokitOptions {
```

Comment:
```suggestion
export interface OctokitThrottlingOptions extends OctokitOptions {
```

Reply:
This is what I tried initially but in `src/test/octokit.ts` I get the following TS Error:

<img width="918" alt="image" src="https://user-images.githubusercontent.com/2574275/155200221-2beaa39e-249f-4097-b6fd-a452301a6ffb.png">